### ISSUE TITLE
Slack error during migration:  `service_removed`

### ISSUE THREAD

**Issue by bkeepers** [integrations/slack#354]
https://sentry.io/github-integrations/slack/issues/471109303/

```
Response not OK:  invalid_service
```

Appears to be coming from: https://github.com/integrations/slack/blob/cad3ad0b0e7d5ab3efa535d5da8c7591786e7c16/lib/models/legacy-subscription.js#L111

We've seen this error > 100 times in the last 24 hours.

cc @bertrandom @wilhelmklopp

**Comment by wilhelmklopp** [comment]
@bertrandom and I were investigating this, so please correct me if I'm wrong here @bertrandom 

This happens in cases where people manually disable their old configuration before they re-subscribe,
so based on the data we initially received from Slack in the `config_migration` event, we still think it's active, we ultimately call Slack to de-activate, but it's already gone so Slack serves us with this `invalid_service` response
TL;DR: it's not really a problem, but we should deal with the error more gracefully and set the legacy subscription as reactivated

**Comment by bertrandom** [comment]
The gist of this is correct, but some details to clarify. With the Slack-owned app, you can either *disable* a configuration or *remove* a configuration. If the user has an existing configuration, installs the Github-owned app, and then proactively removes the configuration before running the subscribe slash command, then the API call will fail with "invalid_service". We should treat this as an expected behavior. If you get other errors from that API method consistently, we'd like to hear about them though, but that one I think should be expected.

**Comment by wilhelmklopp** [comment]
We seem to still get this error in Sentry: https://sentry.io/github-integrations/slack/issues/471827621/

**Comment by wilhelmklopp** [comment]
We're ignoring this issue in Sentry now, which seems like the best solution to work around the node-slack-sdk logging settings
